File: mobile/src/main/java/com/windscribe/mobile/newsfeedactivity/NewsFeedListener.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.newsfeedactivity;

import com.windscribe.vpn.localdatabase.tables.WindNotification;

public interface NewsFeedListener {

    void onNotificationActionClick(WindNotification windNotification);

    void onNotificationExpand(WindNotification windNotification);
}

File: mobile/src/main/java/com/windscribe/mobile/newsfeedactivity/NewsFeedListener.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.newsfeedactivity

import com.windscribe.vpn.localdatabase.tables.WindNotification

interface NewsFeedListener {
    fun onNotificationActionClick(windNotification: WindNotification)
    fun onNotificationExpand(windNotification: WindNotification)
}